src/test/Components/CreateImageWizard/steps/TargetEnvironment/AwsTarget.test.tsx

```diff
@@ -83,6 +99,181 @@ const enterAccountId = async () => {
   await waitFor(async () => user.type(awsAccountIdTextbox, '123123123123'));
 };
 
+let router: RemixRouter | undefined = undefined;
+const routes = [
+  {
+    path: 'insights/image-builder/imagewizard/:composeId?',
+    element: <CreateImageWizard />,
+  },
+];
```

Comment:
Ah, I see. There's a default path to `/insights/image-builder` missing in the routes. Adding it to the AWS and GCP test suite should do the trick :magic_wand: 

```suggestion
const routes = [
  {
    path: 'insights/image-builder/*',
    element: <div />,
  },
  {
    path: 'insights/image-builder/imagewizard/:composeId?',
    element: <CreateImageWizard />,
  },
];
```